Bottom: 8a9c77f483164bbbfd2673cb9bac4b5dca080839
Top:    f76e36f1f0061179aa216410cdfc66e7b54b254c
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-02-16 22:44:00 +1300

Refresh of define-stream-result

---

diff --git a/testtools/testresult/real.py b/testtools/testresult/real.py
index 98cc7d3..22f3b1f 100644
--- a/testtools/testresult/real.py
+++ b/testtools/testresult/real.py
@@ -262,7 +262,7 @@ class StreamResult(object):
     generally make a sequence of calls like::
 
       >>> result.status(self.id(), 'inprogress')
-      >>> result.status(self.id(), 'finish')
+      >>> result.status(self.id(), 'success')
 
     General concepts
     ----------------
